pkg/resource/promtail-config/promtail-config.go

```diff
@@ -68,13 +70,24 @@ func ConfigMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
 // the promtail extra-config
 func GeneratePromtailConfig(lc loggedcluster.Interface) (v1.ConfigMap, error) {
 
+	// Scrape logs from kube-system and giantswarm namespaces only for WC clusters
+	extraRelabelConfigs := `[]`
+	if lc.GetObservabilityBundleConfigMap() == key.WCObservabilityBundleConfigMap {
```

Comment:
As `WCObservabilityBundleConfigMap == CAPIObservabilityBundleConfigMap`, how does this make sure it does not apply to CAPI MCs?

Reply:
Why should it not apply to CAPI MCs though?

Reply:
oh nevermind

Reply:
We need a isWorkloadCluster condition

Reply:
In PMO it's mostly InstallationName == ClusterName